content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13.md

```diff
@@ -209,21 +261,25 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
     node "ip-172-31-85-18" drained
     ```
 
+## Upgrade kubeadm and the kubelet on worker nodes
+
 1.  Upgrade the Kubernetes package version on each `$NODE` node by running the Linux package manager for your distribution:
 
-    {{< tabs name="k8s_install" >}}
+    {{< tabs name="k8s_kubelet_and_kubeadm" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.13.x-00 with the latest patch version
     apt-get update
-    apt-get upgrade -y kubelet kubeadm
+    apt-get install -y kubelet=1.13.x-00 kubeadm=1.13.x-00
     {{% /tab %}}
     {{% tab name="CentOS, RHEL or Fedora" %}}
-    yum upgrade -y kubelet kubeadm --disableexcludes=kubernetes
+    # replace x in 1.13.x-0 with the latest patch version
+    yum install -y kubelet-1.13.x-0 kubeadm-1.13.x-0 --disableexcludes=kubernetes
     {{% /tab %}}
     {{< /tabs >}}
 
-## Upgrade kubelet on each node
+## Upgrade the kubelet config on each worker node
```

Comment:
this is ok I guess, but I think config should be upgraded before the binary, so that when the kubelet restarts with the new version it'll pick up the new config.

Reply:
thanks, updated.